<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recep_WelcomePage</class>
 <widget class="QDialog" name="Recep_WelcomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>842</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ClosedHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Receptionist Welcome Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 223, 255);</string>
  </property>
  <widget class="QPushButton" name="WelcomePage_Guest_Room_Reserved_PB">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 232, 174);</string>
   </property>
   <property name="text">
    <string>See All Rooms a Guest Has Reserved</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Guest_Birthday_PB">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>720</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(24, 255, 178);</string>
   </property>
   <property name="text">
    <string>See Guests who Resided in Hotel on Their Birthday</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Waiting_Reservations_PB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 255, 204);</string>
   </property>
   <property name="text">
    <string>List of Waiting Reservations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_list_of_all_reservations_PB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 134, 12);</string>
   </property>
   <property name="text">
    <string>List of All Reservations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_List_room_status_PB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 151, 196);</string>
   </property>
   <property name="text">
    <string>List of Rooms By Their Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Guest_Bill_PB">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 154, 255);</string>
   </property>
   <property name="text">
    <string>See Guests with Bills over 1,000,000</string>
   </property>
  </widget>
  <widget class="QLabel" name="WelcomePage_header_L">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(201, 204, 255);</string>
   </property>
   <property name="text">
    <string>WELCOME TO HOME PAGE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Cleaning_order_reservation_PB">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 212, 255);</string>
   </property>
   <property name="text">
    <string>Cleaning Orders of a Reservation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Number_Confirmed_Reservation_PB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>590</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 212, 255);</string>
   </property>
   <property name="text">
    <string>Number of Confirmed Reservations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Assign_Staff_PB">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>590</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 212, 255);</string>
   </property>
   <property name="text">
    <string>Assign Staff to Cleaning Order</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
